modules/peridynamics/doc/content/source/userobjects/SingularShapeTensorEliminatorUserObjectPD.md

```diff
@@ -0,0 +1,13 @@
+# SingularShapeTensorEliminatorUserObjectPD
+
+## Description
+
+The `SingularShapeTensorEliminatorUserObjectPD` UserObject is used to eliminate the occurrence of singular shape tensors in peridynamic simulations. This UserObject is executed at end of a timestep, using the updated bond status provided by an AuxKernel. The shape tensor is formed using neighboring bonds (including potentially ghosted bonds from neighboring processor(s)) for each node connected to a bond. Any bbond with at least one singular associated shape tensor is treated as being broken, just as it would be if determined by a bond-based failure criterion. Because breaking bonds can have make other shape tensors singular, this can have a cascading effect, so this process is repeated until no further singular shape tensors are detected.
```

Comment:
bbond -> bond

Reply:
By the way, I made a bunch of edits to this file (missed that one) and added that as a commit. Make sure you work on top of that one (or just copy that file over).